startup_scripts/215_dcim_interfaces.py

```diff
@@ -0,0 +1,46 @@
+from dcim.models import Interface, Device
+from extras.models import CustomField, CustomFieldValue
+from ruamel.yaml import YAML
+
+from pathlib import Path
+import sys
+
+file = Path('/opt/netbox/initializers/dcim_interfaces.yml')
+if not file.is_file():
+  sys.exit()
+
+with file.open('r') as stream:
+  yaml = YAML(typ='safe')
+  interfaces = yaml.load(stream)
+
+  optional_assocs = {
+    'device': (Device, 'name')
```

Comment:
Is this assoc really optional? An interface without a device does not make sense to me, or am I missing anything?